server/lib/files/handle-multipart-files.ts

```diff
@@ -0,0 +1,69 @@
+import { unlink } from 'fs/promises'
+import Koa from 'koa'
+import koaBody from 'koa-body'
+import { FilesErrorCode } from '../../../common/files'
+import { asHttpError } from '../errors/error-with-payload'
+import logger from '../logging/logger'
+import { FilesError } from './files-error'
+
+function convertFormidableError(err: Error) {
+  // TODO(2Pac): Can't do a much better check here since koa-body uses an old version of formidable.
+  // Looks like it might get updated soon tho: https://github.com/koajs/koa-body/issues/241
+  if ('code' in err === false) {
+    throw err
+  }
+
+  switch ((err as any).code) {
+    // TODO(2Pac): koa-body seems to be using an ancient version of formidable internally; in the
```

Comment:
We need a test for this or something, no load-bearing comments please

Reply:
I was actually wrong in my initial assumption and they didn't change the meaning of the error code. They just renamed it to be more accurate, and added a new error code. So now there are error codes if the _total_ files size exceeds the limit, or if any of the files exceed it.

But anyway, I'm handling both here even though only one is used in this old version of koa-body.